src/style.css

```diff
@@ -0,0 +1,146 @@
+.html {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
I think reseting margin here is useless because you already did it in  a body. So delete it unless it's necessary.